usaspending_api/download/helpers.py

```diff
@@ -198,17 +198,29 @@ def split_csv(file_path, delimiter=',', row_limit=10000, output_name_template='o
         `output_name_template`: A %s-style template for the numbered output files.
         `output_path`: Where to stick the output files.
         `keep_headers`: Whether or not to print the headers in each output file.
+        `file_name`: Alternative naming convention for output files (optional)
     Example usage:
         >> from toolbox import csv_splitter;
         >> csv_splitter.split('/home/ben/input.csv');
     """
     split_csvs = []
     reader = csv.reader(open(file_path, 'r'), delimiter=delimiter)
     current_piece = 1
-    current_out_path = os.path.join(
-        output_path,
-        output_name_template % current_piece
-    )
+    if file_name is not None:
+        zip_idx = file_name.find(".zip")
+
+        if(zip_idx > 0):
+            file_name = "%s.csv" % file_name[:zip_idx]
```

Comment:
Can you generate the `file_name` outside of `split_csv`? It seems a little counter-intuitive for a `split_csv` function to take in an argument that's expected to be a `.zip`. Also would recommend using `os` instead of string's `find` for getting the filename sans extension like so (it's your call, just my preference):

```
import os
...
file_name = os.path.splitext("path_to_file")[0]
```

Reply:
Simplified the filename preservation to be one level up, also changed the method to be less rigid.